languages/tree-sitter-stack-graphs-typescript/src/stack-graphs.tsg

```diff
@@ -6160,3 +6194,131 @@ if none @is_acc {
   attr (@async.async_type__promise_ref__ns) push_symbol = "%T"
   edge @async.async_type__promise_ref__ns -> @async.lexical_scope
 }
+
+;
+;       #  #####  #     #
+;       # #     #  #   #
+;       # #         # #
+;       #  #####     #
+; #     #       #   # #
+; #     # #     #  #   #
+;  #####   #####  #     #
+;
+
+; #dialect tsx
```

Comment:
I think this one is missing an #end below. We could at a check to the preprocessing code that we don't end inside a block.